CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,5 @@
 - Fix: idPattern '.*' is now allowed in NGSIv2 registrations (#3458)
 - Add: Forwarded queries work for registrations using idPattern == '.*', but only for simple cases (#3458)
+- Add: Forwarded updates work for registrations using idPattern == '.*', but only for simple cases (#3458)
```

Comment:
This line seems to be duplicated.

Reply:
One line is about queries, the other about updates.
They **are** very similar but also completely different

Reply:
I misses the "queries" vs "udpates" difference. I's ok. Sorry the noise.

NTC